.mailmap

```diff
@@ -22,111 +22,75 @@ Mark Scherer <euromark@web.de> mscherer <mark.scherer@spryker.com>
 Yevgeny Tomenko <skie@mail.ru>
-Yevgeny Tomenko <skie@mail.ru> SkieDr <skie@mail.ru>
 Ricardo Arturo Cabral <ricardo.arturo.cabral@gmail.com>
-Ricardo Arturo Cabral <ricardo.arturo.cabral@gmail.com> Cameri <ricardo.arturo.cabral@gmail.com>
-Robert <r.pustulka@zano.pl>
-Robert <r.pustulka@zano.pl> Robert Pustułka <r.pustulka@zano.pl>
 Jonas <jh@ht-studios.de>
-Jonas <jh@ht-studios.de> ionas <jh@ht-studios.de>
-Jonas <jh@ht-studios.de> Jonas Hartmann <jh@ht-studios.de>
 Cauan Cabral <cauan@radig.com.br>
-Cauan Cabral <cauan@radig.com.br> CauanCabral <cauan@radig.com.br>
 pirouet <pirouet@me.com>
-pirouet <pirouet@me.com> Michael Pirouet <pirouet@me.com>
 davidsteinsland <david@davidsteinsland.net>
-davidsteinsland <david@davidsteinsland.net> David Steinsland <david@davidsteinsland.net>
 jamiemill <jamiermill@gmail.com>
-jamiemill <jamiermill@gmail.com> Jamie Mill <jamiermill@gmail.com>
 Stefan Dickmann <stefan@php-engineer.de>
-Stefan Dickmann <stefan@php-engineer.de> Stefan D <stefan@php-engineer.de>
 Benjamin Tamási <h@lfto.me>
-Benjamin Tamási <h@lfto.me> Tamási Benjamin <h@lfto.me>
 Gordon Pettey (petteyg) <petteyg359@gmail.com>
-Gordon Pettey (petteyg) <petteyg359@gmail.com> Gordon Pettey <petteyg359@gmail.com>
 Mathieu de Ruiter <mathieu@fellicht.nl>
-Mathieu de Ruiter <mathieu@fellicht.nl> Mathieu <mathieu@fellicht.nl>
 Cees-Jan Kiewiet <ceesjank@gmail.com>
-Cees-Jan Kiewiet <ceesjank@gmail.com> wyrihaximus <ceesjank@gmail.com>
 Fiblan <fib@ingegnosamente.com>
-Fiblan <fib@ingegnosamente.com> fiblan <fib@ingegnosamente.com>
 Haithem BEN GHORBAL <haithem.benghorbal@gmail.com>
-Haithem BEN GHORBAL <haithem.benghorbal@gmail.com> Haithem Ben Ghorbal <haithem.benghorbal@gmail.com>
 Pedro Perejón <pperejon@proavan.com>
-Pedro Perejón <pperejon@proavan.com> Pedro Perejon <pperejon@proavan.com>
 Algirdas Gurevicius <a.gurevicius@gmail.com>
-Algirdas Gurevicius <a.gurevicius@gmail.com> uniquexor <a.gurevicius@gmail.com>
 Calin <calinseciu@gmail.com>
-Calin <calinseciu@gmail.com> calinseciu <calinseciu@gmail.com>
 Mikaël Capelle <capelle.mikael@gmail.com>
-Mikaël Capelle <capelle.mikael@gmail.com> mcapelle <capelle.mikael@gmail.com>
-Mikaël Capelle <capelle.mikael@gmail.com> Holt59 <capelle.mikael@gmail.com>
 OKINAKA Kenshin <okinakak@yahoo.co.jp>
-OKINAKA Kenshin <okinakak@yahoo.co.jp> Kenshin Okinaka <okinakak@yahoo.co.jp>
-OKINAKA Kenshin <okinakak@yahoo.co.jp> okinaka <okinakak@yahoo.co.jp>
 Walter Nasich <wnasich@gmail.com>
-Walter Nasich <wnasich@gmail.com> wnasich <wnasich@gmail.com>
 mstra001 <ms@creemedia.com>
-mstra001 <ms@creemedia.com> mathieu strauch <ms@creemedia.com>
 Aymeric Derbois <aymeric@derbois.com>
-Aymeric Derbois <aymeric@derbois.com> Derbois Aymeric <aymeric@derbois.com>
 Daniel <wrightdaniel86@gmail.com>
-Daniel <wrightdaniel86@gmail.com> dangerdan <wrightdaniel86@gmail.com>
 James Michael DuPont <JamesMikeDuPont@gmail.com>
-James Michael DuPont <JamesMikeDuPont@gmail.com> James Michael DuPont <jamesmikedupont@gmail.com>
```

Comment:
Yeah this was a mistake. Do you mind adding it back since you are already doing other changes.